templates/hardhat/solidity/hardhat.config.ts

```diff
@@ -31,17 +32,11 @@ const config: HardhatUserConfig = {
       url: "http://localhost:3050",
       ethNetwork: "http://localhost:8545",
       zksync: true,
-      accounts: process.env.WALLET_PRIVATE_KEY
```

Comment:
why remove the accounts for local nodes? I think we should leave them in

Reply:
yeah, you're right. I removed them cause for local nodes you can use `getSigners`, but I only use that in unit tests, not in deployment scripts...